src/Components/Artwork/__tests__/Details.test.tsx

```diff
@@ -27,44 +28,36 @@ describe("Details", () => {
 
   describe("in sale", () => {
     it("shows 'bidding closed' message if in closed auction", async () => {
-      const data = cloneDeep(artworkInAuction)
+      const data = cloneDeep(mutableArtwork)
       data.sale.is_closed = true
       const wrapper = await getWrapper(data)
       expect(wrapper.html()).toContain("Bidding closed")
     })
 
     it("shows sale 'timely' data", async () => {
-      const data = cloneDeep(artworkInAuction)
-      const wrapper = await getWrapper(data)
-      const html = wrapper.html()
-      expect(html).toContain("ends in 14d")
-    })
-
-    it("shows sale 'timely' data when auction is in a preview state", async () => {
-      const data = cloneDeep(artworkInAuction)
-      data.sale.is_open = false
```

Comment:
`is_open` wasn't used in the component, so I removed it from the query, and that makes these two tests the exact same code. So we nuked it.

Reply:
🔫